service/crt/service.go

```diff
@@ -126,10 +120,15 @@ func New(config Config) (*Service, error) {
 // Boot starts the service and implements the watch for the certificate TPR.
 func (s *Service) Boot() {
 	s.bootOnce.Do(func() {
-		if err := s.tpr.CreateAndWait(); err != nil {
+		err := s.tpr.CreateAndWait()
+		switch {
+		case tpr.IsAlreadyExists(err):
+			s.Config.Logger.Log("info", "certificate third-party resource already exists")
+		case err != nil:
 			panic(fmt.Sprintf("could not create certificate resource: %#v", err))
+		default:
+			s.Config.Logger.Log("info", "successfully created certificate third-party resource")
 		}
```

Comment:
@xh3b4sd I changed if to the above.

My take on that is `err != nil && tpr.IsAlreadyExists(err)` protects from panics.